### PR TITLE
Overhaul main introduction block.

### LINKED ISSUE #122: Make rotating screenshots larger
The screenshots on the main page are tiny! They are actually 500x300 but scaled down.

Here's how it would look with larger screenshots and a new header font:
oraweb

### LINKED ISSUE #241: Add Steam group to community page.
http://steamcommunity.com/groups/openra4life/
is the most well maintained group and should be deemed the official group.

### LINKED ISSUE #249: Suggestion: Connect the web pages
Hey, I noticed that there are alot sub pages and they are not connected enough in my opinion.
I suggest to add a link to gamereplays.org at the resource center
and also to add the resources center at the navigation bar of the main page.
This could boost value of these sub pages.

Can also found here:
https://github.com/OpenRA/OpenRA/issues/8904#issuecomment-127134061

### PR DESCRIPTION
**Pull request by pchote**
Resurrected version of #281 with design iterations by @obrakmann and @x-a-n-a-x.

screen shot 2016-05-13 at 23 52 57

Closes #122.
Closes #241.
Partially resolves #249.

### CODE REVIEW

**Comment by x-a-n-a-x** [comment]
I am not fan of the "Welcome back, Commander" overlay text but I can live with it :)

But for an big overhaul, I would even go further:

The rotating images could turn into animations when the cursor hovers them. 
Example: http://codepen.io/gil--/pen/bNxZWg

If you're "meh" about this concept, note that videos shouldn't have to auto-play on mouseover. Hovering the screenshot could display a  ▶️ PLAY icon.

This way, the frontpage would feature both screenshots (displayed by default) and short gameplay videos of the different mods. This would clearly add motion to static and boring (for some people) screenshots.

File: content/style.css [hunk 1]
```diff
@@ -143,11 +143,23 @@ p.download, .newsDetails {
...
+  border: solid 2px #650b03;
+  border-radius: 10px;
+  -moz-border-radius: 10px;
+  -webkit-border-radius: 10px;
+  -khtml-border-radius: 10px;
+}
+
+#screenshot-banner {
+	z-index: 11;
+    position: absolute;
```
Comment (obrakmann) [inline content/style.css]: Indentation is all over the place in this patch. Can you please fix that?

**Comment by obrakmann** [comment]
Looks otherwise good to me, :+1: 

@x-a-n-a-x: let's not feature-creep and delay this again. We can always add that later.

**Comment by x-a-n-a-x** [comment]
@obrakmann : sure, sure. But what about the IRC icon instead of the Github one ?

**Comment by obrakmann** [comment]
TBH, I think inviting users to our IRC channel directly on the front page isn't a good idea.

The social buttons point to places where users can get up-to-date information on events, new mod releases, things like that. IRC is different to that (well, as is github).

If we had a second #openra-dev channel, that is actually muted, I'd be ok with it. Tech support stuff and balance discussion and so on could then be kept in #openra.

**Comment by x-a-n-a-x** [comment]
ok then :+1:

**Comment by pchote** [comment]
Good point about the css indentation, I'll fix that tonight. I really need to fix my editor to stop breaking that.

**Comment by pchote** [comment]
Added a new commit that fixes style.css.

**Comment by Mailaender** [comment]
The itch.io fallback button seems to have a wrong size. https://github.com/OpenRA/OpenRAWeb/pull/295/files#diff-84921672e6512c7a0d41aaa8c0066d3e

**Comment by HenrytheSlav** [comment]
I often hear on IRC that we should do something more radical and I support that. It still feels that our web is early 2000s era. But this is an improvement so :+1:

**Comment by x-a-n-a-x** [comment]
> The itch.io fallback button seems to have a wrong size.

@Mailaender (that I am going to nickname "eagle-eye" for noticing that ^^) is right. the itch.io fallback image is 54x40 currently.

**Comment by pchote** [comment]
Fixed thanks to @obrakmann.

**Pull request merged**